_minutes/2023-08-17-wecg.md

```diff
@@ -0,0 +1,161 @@
+   * [Issue 440](https://github.com/w3c/webextensions/issues/439): Proposal: add dictionary operations to DNR modifyHeaders
+   * [Issue 208](https://github.com/w3c/webextensions/issues/208): Consider common API for push messages across browsers
+ * **Open discussion queue (add yourself at the bottom)**
+   * &lt;end of list>
+
+
+## Attendees (sign yourself in)
+
+ 1. Rob Wu (Mozilla)
+ 2. Luke Selker (Dashlane)
+ 3. Arthur Coruble (Dashlane)
+ 4. Benjamin Bruneau (1Password)
+ 5. Jason Waterman (Mozilla)
+ 6. Timothy Hatcher (Apple)
+ 7. Kiara Rose (Apple)
+ 8. Richard Worth (Capital One)
+ 9. David Johnson (Apple)
+ 10. Matt Gibson (Bitwarden)
+ 11. Carlos Jeurissen (Jeurissen Apps)
+ 12. Maxim Topciu (AdGuard)
+ 13. Dmitriy Seregin (AdGuard)
+ 14. Oliver Dunk (Google)
+ 15. Mukul Purohit (Microsoft)
+ 16. Jackie Han (No affiliation)
+ 17. Justin Lulejian (Google)
+
+
+## Meeting notes
+
+[Issue 362](https://github.com/w3c/webextensions/issues/362): Declarative cosmetic rules
+
+ * [rob] Andrey (AdGuard) is not here, let's move this to the next meeting.
+
+[Issue 344](https://github.com/w3c/webextensions/issues/344): Regular expressions in DNR API (regexFilter)
+
+ * [rob] Similarly, Andrey also wanted to revisit this topic.
+
+[Issue 154](https://github.com/w3c/webextensions/issues/154): Proposal: browser.secureStorage API
+
+ * [timothy] This API may be a better fit for the web platform. Are there other interested parties?
+ * [oliver] Would be nice to make this a web platform API, which would also result in more people available to work on the implementation across all OSs.
+ * [simeon] Is this connected to WebAuthn?
+ * [oliver] I wanted to check whether relevant Googlers are attending TPAC, but it looks like that is not the case.
+ * [luke] Interest from Dashlane.
+ * [benjamin] Also interest from 1Password. Wondering about details about browser/OS, and how security is guaranteed.
+ * [timothy] We're checking internally (WebKit) how this would or should look like, whether this would or should be a web platform thing.
+
+[Issue 436](https://github.com/w3c/webextensions/issues/436): Freeform inconsistencies
+
+ * [carlos] (see issue) Goal is to reduce inconsistencies of freeform pages.
+ * [rob] Web pages in general or extension UI pages?
+ * [carlos] Extension UI pages (e.g. options UI pages); It could be helpful to define explicitly how these should be handled.
+ * [simeon] A comparison table describing the differences could make it easier to identify actionable parts for the browser.
+ * [carlos] The issue links to a repo with screenshots.
+ * [rob] An explicit comparison with suggestion of desired behavior would make it easier to discuss.
+ * [timothy] Think that this is a good area to focus on. E.g. on WebKit we trigger a special layout mode for autosizing.
+ * [rob] I'm supportive of converging towards something common when it's clear that we should be heading towards one behavior.
+ * [oliver] ...
```

Comment:
@oliverdunk If you recall what you said, could you add it? I recall you expressing some support while also raising concerns, but not the exact message.

Reply:
Here you go:
```suggestion
 * [oliver] In places where the browsers already have similar behaviour, I think it makes sense to align. The one caveat I would call out is that part of our WECG charter allows browsers to differentiate - so we can never rule out a browser deciding to have significantly different rendering.
```